packages/x-data-grid/src/components/menu/columnMenu/GridColumnMenuContainer.tsx

```diff
@@ -32,13 +32,26 @@ const GridColumnMenuContainer = forwardRef<HTMLUListElement, GridColumnMenuConta
       [hideMenu],
     );
 
+    const handleMenuScrollCapture = React.useCallback(
```

Comment:
Nitpick: Since the function doesn’t rely on anything inside the component, maybe it's worth pulling it outside?

Reply:
Done!